### ISSUE TITLE
Outliers to remove from analysis

### ISSUE THREAD

**Issue by kl1986** [KatieA88/CreditScorecard#11]
Just noticed someone with over 100m in income...I suspect we'll need to eliminate some of these

**Comment by KatieA88** [comment]
other than that outlier, the data seems also very skewed to lower earners (< 20k). I guess those are the ones who are most likely to take a loan perhaps?

**Comment by kl1986** [comment]
Yup, i wonder if thats to be expected though? You and i will be in the top 5% of earners so likely no surprise that any population will be be skewed that way. I suspect that it will be even more skewed given its geared towards alternative finance. I had a look at their website and its basically for those with no credit history
